File: Feitos em Python 3/1158.py
vezes=int(input())
z=0
soma=0
while z<vezes:
    vet=input().split()
    x=int(vet[0])
    y=int(vet[1])
    if x%2==0:
        x+=1
        i=0
        while i<y:
            soma+=x
            x+=2
            i+=1
    else:
        i=0
        while i<y:
            soma+=x
            x+=2
            i+=1
    print(soma)
    soma=0
    z+=1

File: Feitos em C Sharp/1158.cs
using System;
namespace ConsoleApp15 {
    class Program {
        static void Main(string[] args) {
			int soma = 0;
			int vezes = int.Parse(Console.ReadLine());
			for (int z = 0; z < vezes; z++) {
				string[] vet = Console.ReadLine().Split();
				int x = int.Parse(vet[0]);
				int y = int.Parse(vet[1]);
				if (x % 2 == 0) {
					x++;
					for (int i = 0; i < y; i++) {
						soma += x;
						x += 2;
					}
				}
				else {
					for (int i = 0; i < y; i++) {
						soma += x;
						x += 2;
					}
				}
				Console.WriteLine(soma);
				soma = 0;
			}
		}
    }
}